### ISSUE TITLE
Reconcile plans for delegated access across MHV/VA.gov

### ISSUE THREAD

**Issue by tmulrooney-va** [department-of-veterans-affairs/va.gov-team#38051]

**Comment by batemapf** [comment]
@MarciMcGuire does this belong in `Current Sprint`? If this work is not detailed elsewhere, I'm happy to take a stab at describing it, or at least the background.

**Comment by MarciMcGuire** [comment]
No, it doesn't belong in the current sprint. Which board are you seeing it in the Current Sprint column?

Sure - if you have time to, that would be great!

**Comment by batemapf** [comment]
Here is where it is listed in current sprint:

![image.png](https://images.zenhubusercontent.com/57d19681e40e5714b16e8696/e6bef172-77b3-4ade-a1aa-aeebb37d0468)

Will move it into backlog and work on building it out a bit.

**Comment by MarciMcGuire** [comment]
Ohh yeah it looks like a lot of things got plopped into Current Sprint in Sept.  I am not sure what happened.

**Comment by MarciMcGuire** [comment]
I'm moving things back to the backlog or ready
